spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Module.scala

```diff
@@ -58,6 +58,20 @@ object Module {
     ModuleLoader.loadFromFile(path)
   }
 
+  /**
+   * Load model from path.
+   *
+   * @param path path to save module, local file system, HDFS and Amazon S3 is supported.
+   *             HDFS path should be like "hdfs://[host]:[port]/xxx"
+   *             Amazon S3 path should be like "s3a://bucket/xxx"
+   * @tparam T numeric type
+   * @return model loaded from path
+   */
+  def loadModuleWithTag[T: ClassTag](path : String)(implicit ev: TensorNumeric[T])
```

Comment:
Don't add a new API; just add an optional argument for storing tag to loadModule.

Reply:
@jason-dai   The tag is not a class field of 'AbstractModule', if we use the same API,  we may want to add that in AbstractModule, shall we ?